src/components/MailSession.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { Dialog, Transition } from '@headlessui/react';
+
+export function MailSession() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  function closeModal() {
+    setIsOpen(false);
+  }
+
+  function openModal() {
+    setIsOpen(true);
+  }
+
+  return (
+    <div className="pb-7">
+      <h3 className="text-4xl font-semibold">Sessão de merenda</h3>
+      <p className="text-2xl pb-4 font-medium">Iniciada às 14:30</p>
+
+      <button
+        className="text-2xl rounded-lg bg-red-600 p-4 pl-10 pr-10 font-bold text-white"
+        onClick={openModal}
+      >
+        Excluir
```

Comment:
O termo correto seria "Encerrar" a sessão de merenda.
```suggestion
        Encerrar
```